### PR TITLE
chore(db): db script to create index on smscounts

### PR DESCRIPTION
**Pull request by seaerchin**
## Problem
querying smscounts is slow, even when we index on `formAdmin` and `msgSrvcId`. This PR aims to solve that by creating a boolean key to index on, so that we can speed up the query.

## Solution
Creates a boolean key in `smscounts` that tracks whether the `msgSrvcSid` used in the verification is formSG's own.

### CODE REVIEW

File: scripts/20210628_create-isOnboardedAcc/create-isOnboardedAcc.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+/*
+This script creates a new key, isOnboardedAccount, on the smscounts db which indexes the msgSrvcId variable
+*/
+
+let formTwilioId = 'insert the form twilio id here'
+
+// == PRE-UPDATE CHECKS ==
+// Count the number of verifications
+db.getCollection('smscounts').count({ smsType: 'VERIFICATION' })
```
Comment (karrui) [inline scripts/20210628_create-isOnboardedAcc/create-isOnboardedAcc.js:11]: we should get a count of the documents with/without our own twilio ID too, so we can check if the updated amounts are exactly the same too

File: src/app/services/sms/sms_count.server.model.ts [hunk 2]
```diff
@@ -34,6 +34,9 @@ const VerificationSmsCountSchema = new Schema<IVerificationSmsCountSchema>({
       required: true,
```
Comment (karrui) [inline src/app/services/sms/sms_count.server.model.ts:34]: this PR should be purely the script and not contain anything else.

I think adding a new schema key (and possibly the programatic setting of whether `isOnboardedAccount`) should be done in a separate PR.

**Review by karrui — CHANGES_REQUESTED** [review]

File: src/app/services/sms/sms_count.server.model.ts (hunk 2 above)
Comment (seaerchin) [reply src/app/services/sms/sms_count.server.model.ts:34]: agreed, will remove this and add it in the follow up PR

File: scripts/20210628_create-isOnboardedAcc/create-isOnboardedAcc.js (hunk 1 above)
Comment (seaerchin) [reply scripts/20210628_create-isOnboardedAcc/create-isOnboardedAcc.js:11]: sure, will do

File: scripts/20210628_create-isOnboardedAcc/create-isOnboardedAcc.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+let formTwilioId = 'insert the form twilio id here'
+
+// == PRE-UPDATE CHECKS ==
+// Count the number of verifications (A)
+db.getCollection('smscounts').count({ smsType: 'VERIFICATION' })
+
+// Count of forms using our twilio acc (B)
+db.getCollection('smscounts').count({
+  smsType: 'VERIFICATION',
+msgSrvcSid: {$eq: formTwilioId},
```
Comment (karrui) [inline scripts/20210628_create-isOnboardedAcc/create-isOnboardedAcc.js:16]: nit: spacing

**Review by karrui — APPROVED** [review]
lgtm

File: scripts/20210628_create-isOnboardedAcc/create-isOnboardedAcc.js (hunk 3 above)
Comment (seaerchin) [reply scripts/20210628_create-isOnboardedAcc/create-isOnboardedAcc.js:16]: thx 4 catch
